pylint/checkers/strings.py

```diff
@@ -188,7 +189,7 @@
     "W1307": (
         "Using invalid lookup key %r in format specifier %r",
         "invalid-format-index",
-        "Used when a PEP 3101 format string uses a lookup specifier "
+        "Used when a PEP 3101 format string uses a lookup specifier "  # pylint: disable=possible-forgotten-f-prefix
```

Comment:
Shouldn't this be on the next line?

Reply:
Probably a problem with pylint but the warning is signaled on this one.